### ISSUE TITLE
Extensibility of messaging features

### ISSUE THREAD

**Issue by coopdanger** [coopdanger/ietf-wg-mimi#20]
@larseggert said:
### Paragraph 5
```
 Modern messaging services commonly support numerous features including plain
 and rich text, delivery notifications, read receipts, replies, reactions,
 presence, and many more. The working group will identify a baseline set of
 messaging features and specify a content format to allow this feature set to be
 implemented interoperably. This format must be usable in the presence of E2EE.
 In defining the format, the working group will seek to reuse existing
 primitives (especially existing semantics) including previously defined message
 headers, MIME types, and URIs where practical.
```
Is this baseline set supposed to be extended over time? If yes, it
would be good to say so here, so that the mechanisms the group will
develop can easily support extensions.

**Comment by coopdanger** [comment]
I think we could say "The working group will identify an extensible baseline set of messaging features ..."

**Comment by larseggert** [comment]
WFM

**Comment by coopdanger** [comment]
Fixed in [5368adc](https://github.com/coopdanger/ietf-wg-mimi/commit/5368adc2a3f316988ed28db6eb27c94befd7a05d).
